reference/CloudPay-API-Specification.yaml

```diff
@@ -1379,6 +1379,112 @@ paths:
 
         The Account Endpoint returns an account summary of the Authenticated User.
 
+        **Note** *Only one of the four authorisation parameters are required*
+    parameters: []
+  /api/v1/account/profile/billing:
+    get:
+      operationId: get-api-v1-account-profile-billing
+      summary: Retrieve the authenticated customer's billing profile
+      tags:
+        - Account
```

Comment:
We're getting 'operation tags must be defined in global tags' warning, AFAIK the tag is used to group operations into one category, so we can specify one tag here OR specify new tags in global tag list - but this second option may cause confusion with same operation in multiple categories.

Reply:
ok I see now difference - should only Account for now